src/main/resources/reference.conf

```diff
@@ -408,32 +408,50 @@ resourceTypes = {
       create_study = {
         description = "Has permission to create study"
       }
+      "share_policy::admin" = {
+        description = "Can grant and revoke a users' admin permission"
+      }
       "share_policy::steward" = {
         description = "Can grant and revoke a users' steward permission"
       }
       "read_policies" = {
         description = "Can read policies"
       }
+      "read_policy::steward" = {
+        description = "Can read the steward policy"
+      }
       "alter_policies" = {
         description = "Can alter policies"
       }
+      "delete" = {
+        description = "Can delete policies/resources"
+      }
     }
-    ownerRoleName = "steward"
+    ownerRoleName = "admin"
```

Comment:
Is this going to have an impact on existing resources?  Will there need to be a migration to change the role of existing owners?

Reply:
we aren't in production yet. so no.